npm/webpack-preprocessor/.eslintignore

```diff
@@ -5,5 +5,4 @@
 **/cypress/fixtures
 **/test/fixtures
 **/_test-output
-/cypress/tests/e2e/compile-error.js
-/examples/use-babelrc/cypress/e2e/spec.cy.js
```

Comment:
We might want to keep these .gitingnore paths around? Stale branches pulling in this change will likely might have these git ignore files laying around and git will suggest we check them in?